### ISSUE TITLE
Accessibility: DOB field is not announced by screen readers

### ISSUE THREAD

**Issue by rhysyngsun** [mitodl/micromasters#1992]
Discovered this testing with Chromevox. If you tab into the Date of Birth field the screen reader just says "Month" / "Day" / "Year". What those values are for is never indicated via screen reader.

**Comment by singingwolfboy** [comment]
This is a limitation of ChromeVox itself. Just as different browsers behave in subtly different ways, different screen readers also behave in subtly different ways. If you test the "Date of Birth" field using VoiceOver on Safari, it is able to correctly read out the `<legend>` element, but ChromeVox is unable to do so.
